### DISCUSSION TITLE
Markdown?

### DISCUSSION THREAD

**Discussion by wisdomlight** [jendrikseipp/rednotebook#776] [Q&A] [answered]
On Ubuntu 24 LTS
RenNoteBook version 2.32
Markdown would be so helpful.
I noticed there's some sort of markdown(up?)  with bold and italics.
Is there a way to have more markdown/up options?
The docs says I can use the txt2tags
how?
thank you

**Comment by srgoni** [comment, marked as answer]
txt2tags is a flavor of markdown that's a bit similar to MediaWiki: https://txt2tags.org/markup.html

There are many other flavors, with GitHub-style markdown being one of the more popular ones: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

There's also an effort to standardize the different flavors: https://commonmark.org/

**Reply by wisdomlight** [reply]
Thank you for his - https://txt2tags.org/markup.html - this is exactly what I was looking for - but some how couldn't find.
Cheers
